### PR TITLE
Feat: Storybook Color Tokens and Webpack Build

### PR DESCRIPTION
**Pull request by usmaniqbal998**
### Things done in this PR

- Storybook upgraded to latest and shifted to vite based builds
- Module bundling moved from rollup to webpack because rollup was not generating css files
- Base color tokens added
- Semantic color tokens added

Screenshot 2023-07-27 at 9 42 40 PM

### CODE REVIEW

File: src/index.ts
Comment (eraymeric) [inline src/index.ts:1]: is this intentional change? Are we still able to use previous components when we remove exporting?

File: src/components/index.ts
Comment (eraymeric) [inline src/components/index.ts:1]: same question with src/index.ts

**Review by eraymeric — CHANGES_REQUESTED** [review]

File: src/index.ts
Comment (usmaniqbal998) [reply src/index.ts:1]: Oh sorry i forgot to update this . Let me update

Comment (usmaniqbal998) [reply src/index.ts:1]: This has been fixed

File: src/components/index.ts
Comment (usmaniqbal998) [reply src/components/index.ts:1]: Fixed

**Review by eraymeric — APPROVED** [review]

**Pull request merged**